src/databricks/labs/ucx/progress/install.py

```diff
@@ -14,7 +14,7 @@
 logger = logging.getLogger(__name__)
 
 
-@dataclass(frozen=True, kw_only=True)
```

Comment:
@asnare : Does this matter to you?

Reply:
It doesn't matter in the technical sense.

I think it mattered more when we had lots of adjacent fields with the same types: keyword arguments allow verification of calls by trivial inspection that the correct value is being passed as the correct argument.